src/client/builtin_scene/materials/web_content.frag

```diff
@@ -8,17 +8,70 @@ in float vInstanceTextureEnabled;
 in vec2 uvs;
 #endif
 
+// SDF rendering uniforms
+uniform vec2 uPlaneDimensions;    // Width and height of the plane in logical units
+uniform vec4 uBorderRadius;       // Border radius for each corner (top-left, top-right, bottom-right, bottom-left)
```

Comment:
Web content uses instanced draw calls, thus the plane dimensions and its border radius should be move to the instance data.